content/pop-recovery.md

```diff
@@ -19,23 +19,52 @@ section: software-troubleshooting
+The Recovery Partition is a full copy of the Pop!\_OS installation disk. It can be used exactly the same as if a live disk copy of Pop!\_OS was booted from a USB drive. The existing operating system can be repaired or reinstalled from the recovery mode. You can perform a refresh install, which allows you to reinstall without losing any user data or data in your home directory. Recovery can also perform a clean install, which resets all OS data.
 
 To boot into recovery mode, bring up the <u>systemd-boot</u> menu by holding down <kbd>SPACE</kbd> while the system is booting, or by holding/tapping any function keys **NOT** used to [Access the BIOS/Boot Menu](/articles/boot-menu) (On non-System76 hardware, try the keys <kbd>F1</kbd> through <kbd>F12</kbd>)
 
-> **NOTE:** These instructions assume Pop!\_OS is the only OS running on your system. If you are booting more than one operating system you may need to change your boot order first, or manually select the Pop!\_OS Disk from your BIOS/Boot menu.
+ **NOTE:** These instructions assume Pop!\_OS is the only OS running on your system. If you are booting more than one operating system you may need to change your boot order first, or manually select the Pop!\_OS Disk from your BIOS/Boot menu.
 
 Once the menu is shown, choose **Pop!_OS Recovery**.
 
 ![systemd-boot](/images/pop-recovery/systemd-boot.png)
 
+## Clean Install
+
+This option erases the current install along with all user files. It reformats the drive partitions and installs the version of Pop!\_OS contained in the Recovery partition.
+
+Steps to back up user-files from a Live Disk/Recovery can be found [here](https://support.system76.com/articles/disaster-recovery)
+
+**NOTE:**
+The Recovery partition OS version will either be the same as the OS version that shipped with your computer, or whichever version to which the Recovery partition has been [updated](#update-recovery-partition)
+
+## Refresh Install
+
+The Refresh Install option allows you to reinstall the OS without losing user account information and data in the home directory. If the `Refresh Install` option is not present on the install screen, one of two things may be true.
+
+1. Your drive is encrypted. The Refresh install option may appear after decrypting the drive. A notice about decrypting the drive will be present above the install options.
+
+2. The OS version in Recovery is older than your current OS version. See the [update instructions](#update-recovery-partition)
```

Comment:
This limitation is no longer present: https://github.com/pop-os/distinst/pull/262 

This would still affect people with outdated Recovery partitions (from before that improvement was made), but since the line is linking to the Recovery update instructions anyway, it might be worth rewording to just say "The Recovery partition is out of date" instead of mentioning/comparing to the OS version.

Reply:
Good note, thanks. I've made this change.